getting-started/templates/systemlink-secrets.yaml

```diff
@@ -114,6 +114,9 @@ userservices:
         - serviceName: "webserver"
           key: ""
           hash: "<SHA512 hash of key>"
+        - serviceName: "nisession"
```

Comment:
Remember to also modify the top-level chart in AzDO

Reply:
Addressed in [AB:1946372](https://ni.visualstudio.com/DevCentral/_workitems/edit/1946372)